code/game/machinery/firealarm.dm

```diff
@@ -285,3 +285,37 @@
 		set_light(l_power = 0.8)
 	else
 		set_light(l_power = 0)
+
+/*
+ * Return of Party button
+ */
+
+/area
+	var/party = FALSE

```

Comment:
I honestly feel like this would be better off as an /area application of the decal component but I think that'd require a minor amount of restructuring so I'll handle that myself at some point.